src/cmd/src/standalone.rs

```diff
@@ -382,10 +393,11 @@ impl StartCommand {
         })
     }
 
-    async fn create_ddl_task_executor(
+    pub async fn create_ddl_task_executor(
```

Comment:
Do we need to make it as `public`?
```suggestion
    async fn create_ddl_task_executor(
```

Reply:
Yes, it's used in another project.